downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -114,6 +124,26 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
 
 #### OpenAPI tool
 
+#### Persist tool
+
+- Introduced a new option to the `persist generate` command to give test datastore. This will generate separate client which can use to mock the actual client.
+    - `--test-datastore` - This is optional and used to indicate the test data store. The possible options are h2 datastore for SQL datastores and inmemory datastore for non-SQL datastores.
```

Comment:
```suggestion
- Introduced a new option to the `persist generate` command to provide a test datastore. This will generate a separate client which can use to mock the actual client. The possible values are `h2` for SQL datastores and `inmemory` for non-SQL datastores.
```